main.py

```diff
@@ -550,21 +555,25 @@ def _send_flow_mods_from_request(self, dpid, command, flows_dict=None):
             f"Send FlowMod from request dpid: {dpid} command: {command}"
             f" flows_dict: {flows_dict}"
         )
-        if dpid:
+        try:
+            if not dpid:
+                self._install_flows(
+                    command, flows_dict, self._get_all_switches_enabled()
+                )
+                return jsonify({"response": "FlowMod Messages Sent"}), 202
+
             switch = self.controller.get_switch_by_dpid(dpid)
             if not switch:
                 return jsonify({"response": "dpid not found."}), 404
-            elif switch.is_enabled() is False:
```

Comment:
I haven't changed the logic, but since I was at it, took the opportunity to flatten these nested logic conditionals.